diffpy/pdfmorph/pdfplot.py

```diff
@@ -133,32 +138,32 @@ def comparePDFs(pairlist, labels=None, rmin = None, rmax = None, show = True,
 
     # Scale the x-limit based on the r-extent of the signal. This gives a nice
     # density of PDF peaks.
-    rlim = rvmax - rvmin
-    scale = rlim / 25.0
+    #rlim = rvmax - rvmin
+    #scale = rlim / 25.0
     # Set a reasonable minimum of .8 and maximum of 1
-    scale = min(1, max(scale, 0.8))
-    figsize = [13.5, 4.5]
-    figsize[0] *=  scale
-    fig = plt.figure(1, figsize = figsize)
+    #scale = min(1, max(scale, 0.8))
+    #figsize = [13.5, 4.5]
+    #figsize[0] *=  scale
+    #fig = plt.figure(1, figsize = figsize)
     # Get the margins based on the figure size
-    lm = 0.12 / scale
-    bm = 0.20 / scale
-    rm = 0.02 / scale
-    tm = 0.15 / scale
-    axes = plt.Axes(fig, [lm, bm, 1 - lm - rm, 1 - bm - tm])
-    fig.add_axes(axes)
+    #lm = 0.12 / scale
+    #bm = 0.20 / scale
+    #rm = 0.02 / scale
+    #tm = 0.15 / scale
+    #axes = plt.Axes(fig, [lm, bm, 1 - lm - rm, 1 - bm - tm])
+    #fig.add_axes(axes)
     plt.minorticks_on()
 
-    plt.plot(rdat, grdat, label = labeldata, marker = 'o', markerfacecolor
-            = 'white', markeredgecolor = 'blue', markersize = 7,
-            markeredgewidth = 0.75)
-    plt.plot(rfit, grfit, label = labelfit, linestyle = 'solid', linewidth =
-            2, color = 'red')
-    plt.plot(rdat, offset*numpy.ones_like(diff), linestyle = '--', linewidth
-            = 1, color = 'black', dashes = (15, 15), aa = False)
+    plt.plot(rdat, grdat, label = labeldata) #, marker = 'o', markerfacecolor
+    #        = 'white', markeredgecolor = 'blue', markersize = 7,
+    #        markeredgewidth = 0.75)
+    plt.plot(rfit, grfit, label = labelfit) #, linestyle = 'solid', linewidth =
+    #        2, color = 'red')
+    plt.plot(rdat, offset*numpy.ones_like(diff)) #, linestyle = '--', linewidth
```

Comment:
we usually like the offset diff curve to be green.  This can be accomplished easily here if we change the order of plotting the numpy.ones and the diff.   

even better, override the color just for the horizontal line to be black.  I am not sure if this will break how the color iterator works, but it should be ok if this is the last plot command.  Just try it.